core/src/test/scala/unit/kafka/admin/ReassignPartitionsCommandArgsTest.scala

```diff
@@ -255,7 +225,7 @@ class ReassignPartitionsCommandArgsTest {
   @Test
   def shouldNotAllowBrokersListWithVerifyOption(): Unit = {
     val args = Array(
-      "--zookeeper", "localhost:1234",
+      "--bootstrap-server", "localhost:1234",
```

Comment:
This is a good change since the test is not specifically testing the legacy mode (so it should use the new mode)  Keep this one

Reply:
ok